mesa/agent.py

```diff
@@ -25,6 +24,8 @@
     from mesa.model import Model
     from mesa.space import Position
 
+from mesa.rng import RandomDescriptor
```

Comment:
`mesa.random` is less surprising than `mesa.rng`. People are used to CPython's `random` module, and `numpy.random`.